zellij-client/src/input_handler.rs

```diff
@@ -173,6 +208,20 @@ impl InputHandler {
             .send(ClientInstruction::Exit(ExitReason::Normal))
             .unwrap();
     }
+
+    // Repeatedly send fake mousehold events, to allow scrolling up/down with mouse while selecting
+    fn start_hold_repeater(&mut self, position: Position) {
+        let mut poller = StdinPoller::default();
```

Comment:
Cool solution :) Since this interfaces with the OS though, it needs to go into os_input_output